### DISCUSSION TITLE
Add auto refresh instructions to the docs

### DISCUSSION THREAD

**Discussion by simonarbel** [microsoft/finops-toolkit#1228] [Ideas]
Hello
Had to scratch my head over way to configure auto refresh in the published dashboards from the storage account and eventually found a solution.
Please update the documentation accordingly as no reference for that.

See attached solution
[Cost summary refresh instructions.docx](https://github.com/user-attachments/files/18274311/Cost.summary.refresh.instructions.docx)

**Comment by flanakin** [comment]
I created issue #1298 for this. @simonarbel can you vote it up, please?
